LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
OK! My question was more on the way of "was there a reason for choosing Apache 2?" and that answer is satisfactory enough.